Doc/library/statistics.rst

```diff
@@ -257,7 +257,7 @@ However, for reading convenience, most of the examples show sorted sequences.
       * "Statistics for the Behavioral Sciences", Frederick J Gravetter and
         Larry B Wallnau (8th Edition).
 
-      * Calculating the `median <https://www.ualberta.ca/~opscan/median.html>`_.
+      * Calculating the `median <https://ist.ualberta.ca/services/tsqs>`_.
```

Comment:
This link is unrelated to median... it seems like a broken link.

Maybe remove the link instead?

@stevendaprano: what do you think?

Reply:
Please remove this link.